mathics/format/asy.py

```diff
@@ -136,6 +137,25 @@ def polygon(points):
 add_conversion_fn(ArrowBox, arrow_box)
 
 
+def arrow3dbox(self, **options) -> str:
+    # l = self.style.get_line_width(face_element=False)
+    pen = asy_create_pens(edge_color=self.edge_color, stroke_width=1)
+
+    return "".join(
```

Comment:
My own take is to shy away from such complicated one-line things like this.

Is the intent to draw two lines one without arrows and then the last one with the arrow? Or does this draw all of the lines and then just an arrow head? 

My understanding is the former not the latter. But a comment making that explicit would be great. 

Also why `self.lines[-2:-1]` as opposed to `self.lines[-1]` which is simpler?

Reply:
> My understanding is the former not the latter. But a comment making that explicit would be great.

Yes, I'll add the comment. It's doing the following:
- draw lines between all points except the last;
- draw an arrow between the penultimate and the last point.

> Also why `self.lines[-2:-1]` as opposed to `self.lines[-1]` which is simpler?

None of them are right, it should be `self.lines[-2:]`, my mistake.